subsys/dfu/Kconfig

```diff
@@ -19,6 +19,14 @@ config DFU_TARGET_MCUBOOT
 	help
 	  Enable support for updates that are performed by MCUboot.
 
+config DFU_TARGET_MCUBOOT_SAVE_PROGRESS
+	bool "MCUBoot store write progress to flash"
+	depends on DFU_TARGET_MCUBOOT
+	depends on SETTINGS
+	help
+	  Enabling this will allow dfu_target_mcuboot to store the current write
```

Comment:
Enable this option to allow dfu_target_mcuboot to store the current write progress to flash. In case of power failure or device reset, the operation can then resume from the latest state.

Reply:
Added